core/pkg/evaluator/json.go

```diff
@@ -100,19 +98,10 @@ func (je *JSON) GetState() (string, error) {
 }
 
 func (je *JSON) SetState(payload sync.DataSync) (map[string]interface{}, bool, error) {
-	_, span := je.jsonEvalTracer.Start(
```

Comment:
this traces actually do not offer a lot of value - it is a short span which offers no insights at all. the new approach spans over new data received till sending, and even enriches on the sync server messages with the information, and this should also help for rpc evaluations